compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -640,6 +640,9 @@ error.object.type.not.allowed=\
 error.table.cannot.be.created.without.constraint=\
   table cannot be created without a constraint
 
+error.table.constraint.must.be.a.record=\
+  table type constraint must be a record type
```

Should type come after constraint since we are saying record type
```suggestion
  table constraint type must be a record type
```